iceoryx_hoofs/testing/include/iceoryx_hoofs/testing/watch_dog.hpp

```diff
@@ -68,8 +69,8 @@ class Watchdog
                 .and_then([&](auto& result) {
                     if (result == iox::posix::SemaphoreWaitState::TIMEOUT)
                     {
-                        std::cerr << "Watchdog observed no reaction after " << m_timeToWait << ". Taking measures!"
```

Comment:
Although there is a branch to set the terminate handler and flush the logger when `terminate` is called, I'm leaning towards writing directly to the console in this case. One might still set a new terminate handler in a test and then this important information would not show up.